src/lib.rs

```diff
@@ -2,6 +2,8 @@ pub mod model;
 pub mod optimize;
 pub mod parse;
 pub mod rewrites;
+pub mod input;
+pub mod resnet50;
```

Comment:
Why should resnet50 be in lib file?

Reply:
You must list all top-level modules in lib.rs.

Reply:
Yeah, I have to put it here to use things in resnet50.rs from other files. Is there a nice way to say have a top-level module 'network', but the functions in network are defined across different files like resnet50.rs, rnn.rs?